spec/rails_helper.rb

```diff
@@ -133,6 +134,8 @@
     junit = "tmp/junit/spec-#{[ENV['CIRCLE_NODE_INDEX'], Process.pid].compact.join('-')}/spec.xml"
     config.add_formatter RspecJunitFormatter, junit
   end
+
+  config.include ActiveSupport::Testing::TimeHelpers
```

Comment:
Maybe we need to add `after(:each)` block to `#travel_back`?

Reply:
Why?, time is restored after tests automatically

Reply:
How does it work? I suggested this because I didn't see how it happens.

Reply:
It's done here: https://github.com/rails/rails/blob/8015c2c2cf5c8718449677570f372ceb01318a32/activesupport/lib/active_support/testing/time_helpers.rb#L70 
Including the module it's enough to get `travel_back` called after every test. I checked it with the debugger and `after_teardown` is called.

Reply:
@jlledom , did you try out rspec or minitest. I don't see any references to `after_teardown` in rspec, that's why I asked in the first place.